src/compute/src/render/reduce.rs

```diff
@@ -1573,7 +1588,7 @@ where
                                 }
                             }
                             _ => panic!(
-                                "Unexpected accumulation (aggr={:?}, accum={accum:?}) in dataflow {debug_name}",
+                                "Unexpected accumulation (aggr={:?}, accum={accum:?})",
```

Comment:
Is it not useful to log the dataflow name here too?

Reply:
I tried to keep consistency here with other parts of `reduce.rs` where we do not log the dataflow name on panics that we expect to catch during development, e.g.: https://github.com/MaterializeInc/materialize/blob/0ad20599b3a33cac86618a19d03ee33627cfa903/src/compute/src/render/reduce.rs#L1339
We can create copies of the debug name to send around to these other `panic!` calls, but if we have a complete datatype / aggregate function mismatch here, we are in much deeper trouble than with invalid accumulations.